graph/resolvers/util.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Decorates the typeResolver with the tags field.
+ */
+const decorateWithTags = (typeResolver) => {
+  typeResolver.tags = ({tags = []}, _, {user}) => {
+    if (user && user.hasRoles('ADMIN')) {
```

Comment:
This should be ADMIN || MODERATOR atm. Are we going to a user.canDo('seeTags') type of scheme to centralize all role logic on the backend or does it make the most sense to explicitly hardcode it here?